serde/src/test/kotlin/com/amazon/ionhiveserde/objectinspectors/IonStructToMapObjectInspectorTest.kt

```diff
@@ -59,26 +60,40 @@ class IonStructToMapObjectInspectorTest {
         assertNull(subject.getMapValueElement(makeStruct(), null))
     }
 
+    @Test
+    fun getMapSize() {
+        assertEquals(3, subject.getMapSize(makeStruct()))
+    }
```

Comment:
This test makes me ask "WHY is 3 the right value?". Inlining `makeStruct()` will make the answer obvious.